examples/language-modeling/eval/__init__.py

```diff
@@ -0,0 +1,5 @@
+if __name__ == "__main__":
+    import sys
+    sys.path.insert(0, './')
```

Comment:
make sure the path is correct, we assume users are in langugae-modeling folder, and call python3 eval/evaluation.py